### DISCUSSION TITLE
[Modification] - Link Menu

### DISCUSSION THREAD

**Discussion by adamdot** [Legal-and-General/canopy#1129] [Canopy requests]
**Type of request**
Modification

**Component, pattern, template, helper or foundation?**
Component

**Nature of request**
We need to build a Link list component for Dotcom-platform but we also need to be able to swap the chevron for different icons and we need to be able to align the icons to the left or the right of the link area.

I don't have screenshots showcasing the new icons.

**Comment by alexcanning** [comment]
Could we get some further detail on what the requirement is? Are there any supporting designs that indicate how this modification will be used?

**Reply by paul-request** [reply]
Yeah, would be good to see what the differences are between the current link list and the proposed changes and maybe the rationale from the designer as to why the changes are necessary as opposed to using the current link list component

**Reply by adamdot** [reply]
After speaking with James Hayden-Ellis, it seems there was a mix up on our end and we don't need to use different icons so this is probably alright how it is. Sorry for the dud discussion! We can close this.

**Reply by elenagarrone** [reply]
Perfect thank you! will mark this as resolved 👍
